home-page.html

```diff
@@ -11,11 +11,63 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
```

Comment:
- [ ] Great job so far but I think in your about page for the desktop version it would be great if you fix spaces here so please decrease the margin/padding between the elements since your design it should be identical to the original design so please fix the spaces here according to the requirements 🤩🔥♻🙏🏻✨

- Your design
![image](https://user-images.githubusercontent.com/58553711/219921896-d873226f-4746-4132-994f-530c72d4d801.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/219921927-e414d064-202b-4bc7-8cd6-5d50c7c6f83c.png)